### PR TITLE
Add Changelog starting with 1.4.0

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+--------------------------------------------------------------------------------
+
+
+[1.5.0] (2019-09-12)
+--------------------
+
+### Improvements
+
+* Check now also individual `WebElement`s, making recheck a "Jest for Java".
```
Comment (beatngu13) [inline CHANGELOG.md]: Hmmm, snapshot testing is just a small part of Jest. It also has a test runner, mocking capabilities, coverage tracker, … moreover, difference testing is different than snapshot testing as we don't diff text. You really wanna keep it? I find it a bit confusing.

**Comment by beatngu13** [comment]
Did you forget to add a reviewer?

File: CHANGELOG.md (hunk 1 above)
Comment (roesslerj) [reply CHANGELOG.md]: If someone enters "Jest for Java" as a search term, I want recheck to show up. There are differences, but still... I think in the Changelog, it doesn't hurt...

Comment (beatngu13) [reply CHANGELOG.md]: Hmmm, I don't think that the changelog is the right place to do SEO … here we want to give clear information to our users what has changed, no marketing blabber.

Just my two cents, feel free to merge anyway.

**Pull request merged**